content/en/platform/corda/5.0/deploying-operating/config-users/permission-system-entities.md

```diff
@@ -0,0 +1,86 @@
+---
+date: '2023-04-24'
+title: "Permission System Entities"
+menu:
+  corda-5:
+    parent: corda-5-cluster-config-users
+    identifier: corda-5-cluster-permission-system-entities
+    weight: 3000
+section_menu: corda-5
+---
+
+The R3's role-based access control (RBAC) permissions system is made up of multiple entities.
+The following section describes those entities and relationships between them.
+
+## User
+
+This entity represents a human user or system user.
+
+### Properties
```

Comment:
```suggestion
### Attributes
```
I would make such a rename to avoid confusion with `Property` entity that we have as part of RBAC entity model.